internal/platform/web/web.go

```diff
@@ -95,3 +99,25 @@ func (a *App) Handle(verb, path string, handler Handler, mw ...Middleware) {
 	// Add this handler for the specified verb and route.
 	a.TreeMux.Handle(verb, path, h)
 }
+
+// clientIPFromRequest implements effort to return the real client IP
+func clientIPFromRequest(r *http.Request) string {
+	// It parses X-Real-IP and X-Forwarded-For in order to work properly with reverse-proxies such us: nginx or haproxy or AWS ALB/LB
+	// Use X-Forwarded-For before X-Real-Ip as nginx uses X-Real-Ip with the proxy's IP.
+	clientIP := r.Header.Get("X-Forwarded-For")
+	clientIP = strings.TrimSpace(strings.Split(clientIP, ",")[0])
```

Comment:
What is the split has len = 0?

Reply:
I worried about that too but I checked. Split always returns a slice with len >= 1 (unless the substring is blank for some reason but ours isn't, it's `","`).

Reply:
Thanks!